src/docker.zsh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env zsh
+
+autoload -U colors
+colors
+
+_fzf_complete_docker() {
+    local subcommand=${${(Q)${(z)@}}[2]}
+
+    if [[ $subcommand =~ (create|history|run) ]]; then
+        _fzf_complete_docker-images '' $@
+        return
+    fi
+
+    if [[ $subcommand =~ (rmi|save) ]]; then
+        _fzf_complete_docker-images '--multi' $@
+        return
+    fi
+}
+
+_fzf_complete_docker-images() {
+    local fzf_options=$1
+    shift 1
+
+    _fzf_complete "--ansi --tiebreak=index --header-lines=1 $fzf_options" $@ < <(
+        docker images --format 'table {{.Repository}};{{.Tag}};{{.ID}};{{if .CreatedSince }}{{.CreatedSince}}{{else}}N/A{{end}};{{.Size}}' 2> /dev/null \
+            | FS=';' _fzf_complete_docker_tabularize $fg[yellow] $reset_color{,,}
```

Comment:
Move the image ID to the first column and always complete that instead of the repository. Consistency is important.